### ISSUE TITLE
Python file

### ISSUE THREAD

**Issue by Ricky-Tigg** [jtesta/ssh-audit#145]
(output truncated)
```
$ pip3 show -f ssh-audit
Version: 2.5.0
License: UNKNOWN
Location: /usr/local/lib/python3.10/site-packages
Requires: 
Required-by: 
Files:
  ../../../bin/ssh-audit
  ssh_audit-2.5.0.dist-info/LICENSE
```
Hello. Be it unknown or not to Python, the license is dully mentioned here
```
$ cat /usr/local/lib/python3.10/site-packages/ssh_audit-2.5.0.dist-info/LICENSE | awk 'NF'| head -2
The MIT License (MIT)
Copyright (C) 2017-2020 Joe Testa (jtesta@positronsecurity.com)
```
Whatever the side the issue has root, it is worth fixing. Copyright's end-year might require up-to-date number. Worth noting that an **en-dash** (–), is what the usage defines as applying to range of dates, not a **hyphen** (-), which is commonly misused by developers.

**Comment by jtesta** [comment]
@Ricky-Tigg : Thanks for bringing this up.  I'll take a closer look next time I poke around with the PyPI packaging.

**Comment by jtesta** [comment]
Fixed in the v2.9.0 release.  Thanks @Ricky-Tigg for reporting this!
